src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -754,6 +777,11 @@ private static class CliOptions {
                         + "be specified.")
         private boolean printTreeWithJavadoc;
 
+        /** Option that controls whether to print the custom AST of the file. */
+        @Option(names = {"-pc", "--printWithCustomTree"},
```

Comment:
A multi-character short option may be confusing, see
https://picocli.info/#_short_posix_options 

Although the library supports multi-character short names, it collects them into groups:
```bash
  -o=<outputPath>           Sets the output file. Defaults to stdout.
  -p=<propertiesFile>       Sets the property files to load.
      -pc, --printWithCustomTree=<customTreePrinterClass>
                            Prints Tree based on the provided custom class.
```
It looks like `-pc` is an extension to `-p`. We can change it to `-tc` so the usage will be

```bash
  -t, --tree                Prints Abstract Syntax Tree(AST) of the checked file. The option cannot
                              be used other options and requires exactly one file to run on to be
                              specified.
  -T, --treeWithComments    Prints Abstract Syntax Tree(AST) with comment nodes of the checked
                              file. The option cannot be used with other options and requires
                              exactly one file to run on to be specified.
      -tc, --printWithCustomTree=<customTreePrinterClass>
                            Prints Tree based on the provided custom class.
```

Reply:
I am fine with this change, but just asking if there is someway we can change `printWithCustomTree` to line up with `tc`? Maybe `treeWithCustomPrinter`?

On a side note, you don't think it may be confused with `treeWithComments`?

Reply:
Yes, it is confusing. Other options may be: `-C` or no short option at all.

Reply:
I removed the short. I was going down that path before, but the `CliOptionsXdocsSyncTest` was still picking it up as a short option even though it wasn't short at all. I changed the code there.